enterprise/admin/forms.py

```diff
@@ -86,7 +87,22 @@ def clean_email_or_username(self):
             return email_or_username
 
         email = email_or_username__to__email(email_or_username)
-        validate_email_to_link(email, email_or_username, ValidationMessages.INVALID_EMAIL_OR_USERNAME)
+        bulk_entry = len(split_usernames_and_emails(email)) > 1
+        if bulk_entry:
+            for email in split_usernames_and_emails(email):
+                validate_email_to_link(
+                    email,
+                    None,
+                    ValidationMessages.INVALID_EMAIL_OR_USERNAME,
+                    ignore_existing=True
+                )
+            email = email_or_username
+        else:
+            validate_email_to_link(
```

Comment:
Again, good call. The `split_usernames_and_emails` call results in a single-item iterable, so we can just iterate across that.